update.xml

```diff
@@ -276,7 +276,26 @@
     <!-- LYF -->
 
     <!-- MOTOROLA -->
-    <manufacturer id="Motorola"> 
+    	 <manufacturer id="Motorola">
```

Comment:
manufacturer Motorola exists, don't add a new one
add your device lower in the file